devise-security.gemspec

```
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
     'Alexander Dreher',
     'Nate Bird',
     'Dillon Welch',
-    'Kevin Olbrich'
+    'Kevin Olbrich',
```

```suggestion
    'Kevin Olbrich'
```

It looks like this may have been autocorrected. It's probably fine to leave it in, but I figured I'd suggest keeping it as it was 🤷

---

@tmr08c Sure, Will add a new commit for that, Thanks.

---

@tmr08c  Added the commit now, please check.